### DISCUSSION TITLE
How to disable pagination dots while using centeredSlides Mode?

### DISCUSSION THREAD

**Discussion by Dani-Boy92** [nolimits4web/swiper#5040] [Q&A]
I use a swiper as an infinite scroller that automatically fetches more items.  I would like for my case to disable the pagination points in the bottom as it can quickly be a lot if there as more items being fetched. I have already looked in the documentation but I have not found out how to turn off these pagination points. The only option i see is to turn `centeredSlides={false}`. However i need centered Slides.

I am thankful for each hint!

**Comment by FelipeRibeiroLabs** [comment]
The dots are activated when you use the pagination module, remove it from your code and the dots will no longer appear!

**Comment by Arhell** [comment]
https://swiperjs.com/swiper-api#pagination
or remove with css `displays: none`
